src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -970,6 +973,11 @@ def customizePerJob(self, job):
         if job.get('inputDatasetLocations'):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
+        # HighIO jobs
+        highIOjob = job.get('highIOjob')
+        if highIOjob:
+            jdl.append('+Requestioslots = 1\n')
```

Comment:
I have no idea actually what it'll do if there are multiple sites it can match, some with and some without highIO job slots. In practical terms, if you look at the job types this is active for, the possible sites they can run at is only a single site anyways, so not sure it matters what the exact behavior is. At least I can't think of a reason why we would care where it matches first in such a case.